libs/features/components/banners/src/lib/elewa-group-team-member/elewa-group-team-member.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="banner-container">
+    <ng-container *ngFor="let banner of banners; index as i">
+        <div class="banner" *ngIf="i === currentBannerIndex">
+            <div class="banner-image" [style.background-image]="'url(' + banner.imageUrl + ')'"></div>
+            <div class="banner-info">
+              <h1>{{banner.name}}</h1>
+              <h2>{{banner.title}}</h2>
+              <p>{{banner.description}}</p>
+              <div class="linkedin-icon">
+                <a [href]="banner.linkedInUrl"><i class="fa-brands fa-linkedin-in">LinkedIn</i></a>
+              </div>
+            </div>
+            <div class="close-button-container">
+              <button class="close-button">
+               <!-- <i class="fa-thin fa-xmark-large"></i> best design -->
+               <button class="close-button" (click)="goBackToMultiUserView()">
+                <i class="fa-solid fa-xmark"></i>
+               </button>
+               
+               
+              </button>
+            </div>
+        </div>
+      </ng-container>
+    <div class="banner-controls">
```

Comment:
This entire component should be moved into a components folder inside the lib components/elewa-group-team-member